.github/workflows/pr-checks.yml

```
@@ -0,0 +1,260 @@
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          cache: 'npm'
+          
+      - name: Install dependencies
+        run: npm ci
+        
+      - name: Check formatting
+        run: npm run format:check
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)